### ISSUE TITLE
[Video] Mode Spray Dapp Showcase with Call to Action to Attract Builders to the Development Ecosystem

### ISSUE THREAD

**Issue by wolfcito** [mode-network/mode-cooperators#78]
## Task Description:
Mode Spray Dapp Enhancement

Enhance the existing Mode Spray Dapp to improve user experience and functionality. The goal is to make the dispersal of crypto more efficient and user-friendly.

It's a complementary task related to #31 

## Acceptance Criteria:
- Create a short video introducing the benefits of using Mode Spray.
- Clearly explain how to use Mode Spray in a user-friendly manner.
- Include a call to action encouraging users to utilize Mode Spray and join Mode's social networks.
- Incorporate the Mode brand into the video for consistent branding.
- Provide an open invitation for contributors to join the development ecosystem

## Resources and References:
Link to Mode Spray Dapp repository: [Mode Spray GitHub](https://github.com/wolfcito/mode-spray)
Link to Mode Spray Dapp on fleek: https://noisy-fish-billions.on-fleek.app

**Comment by wolfcito** [comment]
@joshuanwankwo @fede2442

**Comment by wolfcito** [comment]
Cool! @joshuanwankwo 
I started with the Spanish version! 
https://www.youtube.com/watch?v=mzz1P-K1peA

**Comment by wolfcito** [comment]
Cool! @joshuanwankwo
I have continued with English version
Check it out here: https://youtu.be/pw_QVDLGMW8

**Comment by joshuanwankwo** [comment]
Awesome job man!

Please can you raise a PR and add all the tasks you did in January?

Thank you!

**Comment by wolfcito** [comment]
Right, here is my PR https://github.com/mode-network/mode-cooperators/pull/84
